dev-requirements.txt

```diff
@@ -0,0 +1,6 @@
+wheel
+pylint
+black
+mypy
+types-PyYAML
+docutils-stubs
```

Comment:
[typeshed has docutils stubs](https://pypi.org/project/types-docutils/) included. Not sure which is a better requirement though.

Reply:
In breathe, they had to pin to a version that was consistent with sphinx requirements, but sphinx may have changed their reqs since then
```
types-docutils>=0.14,<0.18
```

Reply:
docutils-stubs is by the maintainer of sphinx and I think is more up to date than typeshed, but not sure

Reply:
After a closer look at typeshed repo, I doubt we can expect any updates to `types-docutils>=0.14,<0.18` stubs. It probably better to use the one maintained for/by sphinx devs